robottelo/api/assertions.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import re
+
+from robottelo.test import AssertApiNotRaisesContextManager
+
+
+def assert_api_not_raises(expected_exception, callable_obj=None,
```

Comment:
This looks like the same code for the CLI exception. Should this become a base class instead?

Reply:
I could extract the code and just pass a proper Assert*NotRaisesContextManage instead of duplication code and using partial. But I thought it would make code a little bit difficult for not enough reason. But I can do that if you prefer.